base/docs/helpdb/Base.jl

```diff
@@ -978,10 +978,12 @@ references in `r` are replaced with the corresponding matched text.
 replace
 
 """
-    randexp([rng], [dims...])
+    randexp([rng], [F], [dims...])
+
+Generate a random number of type `F` according to the exponential distribution with scale 1.
+The type `F` can be `Float16`, `Float32` or `Float64` and defaults to the latter.
```

Comment:
Ok, I updated. I chose to keep the enumeration list, as as a user I would feel frustrated to read "floating-point type" and not to be able to give e.g. `BigFloat`. we could add "currently" to convey the intended future extensions.